src/mysql.c

```diff
@@ -60,6 +60,7 @@ struct mysql_database_s /* {{{ */
 
   bool primary_stats;
   bool replica_stats;
+  bool mariadb;
```

Comment:
same here then

Reply:
[test.zip](https://github.com/collectd/collectd/files/12426226/test.zip)
docker-compose scenario for test mariadb/vanilla